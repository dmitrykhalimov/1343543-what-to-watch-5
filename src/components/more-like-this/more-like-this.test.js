import React from "react";
import renderer from "react-test-renderer";
import MoreLikeThis from "./more-like-this";

const MockComponent = () => {
  return (
    <div>
    </div>
  );
};

describe(`MoreLikeThis`, () => {
  it(`Should MoreLikeThis render correctly`, () => {
    const tree = renderer
      .create(
          <MoreLikeThis>
            <MockComponent/>
          </MoreLikeThis>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
