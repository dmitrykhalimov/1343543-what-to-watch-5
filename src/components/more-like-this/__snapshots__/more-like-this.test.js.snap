// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoreLikeThis Should MoreLikeThis render correctly 1`] = `
<section
  className="catalog catalog--like-this"
>
  <h2
    className="catalog__title"
  >
    More like this
  </h2>
  <div />
</section>
`;
